### PR TITLE
fiona/ replace quil5 image with a new image

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — APPROVED** [review]

File: src/pages/besquare/components/sections/_why-deriv.js [hunk 1]
```diff
@@ -50,18 +50,15 @@ const WhyDeriv = () => {
...
+                    international exposure by collaborating with our overseas offices.
+                    You&apos;ll get the chance to work on exciting projects with
+                    state-of-the-art technologies in an inspiring environment.
                 </TextWrapper>
                 <ImageWrapper
-                    data={data['quil5']}
-                    alt="Quill-5 Deriv's new headquarter in Malaysia"
-                    grid_area="quil5-image"
+                    data={data['bpo_team']}
+                    alt="business process operations team"
```
Comment (yashim-deriv) [inline src/pages/besquare/components/sections/_why-deriv.js:60]:
```suggestion
                    alt="diverse and talented teams"
```
Minor suggestion, the alt should give context to the image.

Comment (fiona-deriv) [reply src/pages/besquare/components/sections/_why-deriv.js:60]: @yashim-deriv  well noted, the image itself was of a bpo team I think

Comment (yashim-deriv) [reply src/pages/besquare/components/sections/_why-deriv.js:60]: then no problem if it fits the current context 👍🏼

**Review by yashim-deriv — DISMISSED** [review]

File: src/pages/besquare/components/sections/_why-deriv.js (hunk 1 above)
Comment (fiona-deriv) [reply src/pages/besquare/components/sections/_why-deriv.js]: it's okay, just implemented your suggested changes

**Review by ShuvoHabib — DISMISSED** [review]

**Review by yashim-deriv — DISMISSED** [review]

**Review by arome-binary — APPROVED** [review]

**Review by kevinw-binary — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**